src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__days"></div>
```

Comment:
Each day should have a unique class 'calendar__day' instead of 'calendar__days'.